### DISCUSSION TITLE
Set the offset for each frame image in AnimatedSprite2D

### DISCUSSION THREAD

**Discussion by xiaoliang6669** [godotengine/godot-proposals#8325] [2D]
“Setting an individual offset for each frame in AnimatedSprite2D, rather than sharing a single offset across all frames, can be beneficial. The offset attribute in the panel could be modified to an array representing the offset of each frame. This is particularly useful for animations with large movements, as it allows for space-saving frame animations. Dynamically controlling the offset position of each frame is the best choice. Although there are other ways to achieve this, they tend to be more complicated. Adding this feature to AnimatedSprite2D would be the optimal solution.”

**Comment by Calinou** [comment]
Thanks for the proposal! Consolidating in https://github.com/godotengine/godot-proposals/issues/2060.
